# Car parameters
wheelBase: 0.248
wheelRadius: 0.0335

# Parameters actuators
offset_angle_servo: 89
min_angle_servo: -20
max_angle_servo: 20
max_speed: 0.45

# Gain controllers
# para 0,3 mm/s
# K_1: -30.3491
# K_2: -9.4208

# para 0,45 mm/s
K_1: -14.0055   
K_2: -6.2294

# Parameters path
A: -1
B: 1
C: 0